### PR TITLE
moolticute: 0.55.0 -> 1.01.0. hughobrien as maintainer.

### PR DESCRIPTION
**Pull request by hughobrien**
###### Description of changes

[Official Release Notes](https://github.com/mooltipass/moolticute/releases/tag/v1.01.0)

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by tomodachi94** [comment]
@hughobrien you might want to reword some of those commit messages to look like this:
```
moolicute:  0.55.0 -> 1.00.1
moolicute: add hughobrien as maintainer
```
Also, the section in the manual on [contributing to Nixpkgs](https://nixos.org/manual/nixpkgs/unstable/#id-1.6) might be handy to make sure your contribution is up-to-spiff before a full review.

**Review by tomodachi94 — APPROVED** [review]
Other than the things mentioned in the comment above, this looks wonderful.

**Comment by tomodachi94** [comment]
Btw, you can use `git rebase -i HEAD~3` to reword those commits; change the command next to the two commits in this PR to `reword`. Your `$EDITOR` will open up so you can edit each message. After you're done, you'll need to do `git push -f` to update the branch.

**Comment by hughobrien** [comment]
> Btw, you can use `git rebase -i HEAD~3` to reword those commits; change the command next to the two commits in this PR to `reword`. Your `$EDITOR` will open up so you can edit each message. After you're done, you'll need to do `git push -f` to update the branch.

very helpful, thank you, and thank you additionally for your kind approach.

File: pkgs/applications/misc/moolticute/default.nix [hunk 1]
```diff
@@ -40,7 +40,11 @@ mkDerivation rec {
     '';
     homepage = "https://github.com/mooltipass/moolticute";
     license = licenses.gpl3Plus;
-    maintainers = [ maintainers.kirikaza ];
+    maintainers =
+      [
+        maintainers.kirikaza
+        maintainers.hughobrien
+      ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/moolticute/default.nix:47]:
```suggestion
    maintainers = with maintainers; [ kirikaza hughobrien ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
please rebase on master

**Comment by hughobrien** [comment]
closing in favour of https://github.com/NixOS/nixpkgs/pull/215034

**Pull request closed without merging**